app/routes/circulars/index.tsx

```diff
@@ -263,15 +270,39 @@ export default function () {
               </li>
             ))}
           </ol>
-          {totalPages > 1 && (
-            <Pagination
-              query={query}
-              page={page}
-              totalPages={totalPages}
-              startDate={startDate}
-              endDate={endDate}
-            />
-          )}
+          <div className="display-flex flex-row flex-wrap">
+            <div className="display-flex flex-align-self-center margin-right-2 width-auto">
+              <div>
+                <Select
+                  id="limit"
+                  className="width-auto height-5 padding-y-0 margin-y-0"
+                  name="limit"
+                  defaultValue="100"
+                  form="searchForm"
+                  onChange={({ target: { form } }) => {
+                    submit(form)
+                  }}
+                >
+                  <option value="10">10 / page</option>
+                  <option value="20">20 / page</option>
+                  <option value="50">50 / page</option>
+                  <option value="100">100 / page</option>
+                </Select>
+              </div>
+            </div>
+            <div className="display-flex flex-justify-center flex-fill flex-align-self-center">
```

Comment:
In the event that we want to left justify the pagination

```suggestion
            <div className="display-flex flex-fill">
```